draft-ietf-rats-eat.md

```diff
@@ -1099,12 +1103,53 @@ string naming the submodule. No submodules may have the same name.
 {::include cddl/submods.cddl}
 ~~~~
 
+
 # Endorsements and Verification Keys {#keyid}
 
 TODO: fill this section in. It will discuss key IDs, endorsement ID and such that
 are needed as input needed to by the Verifier to verify the signature. This will
 NOT discuss the contents of an Endorsement, just and ID/locator.
 
+
+# Layered Attestation {#layered}
+
+It is possible for a device to have multiple Attesting Environments as described in {{RATS-Architecture}}.
+When these are layered, one is a root the endorsement and/or verification key originates with the device manufacturer or endorser.
+The trust for those that are layered above the root comes from Attestation Evidence about them that is provided by the root or a layer below.
+
+A layered Attesting Environment is always described as a submodule.
```

Comment:
so... initially there is the root Attesting Environment and state transitions then go to layered Attesting Environments? I kinda like these words. Will bring that up in the arch team.

Reply:
1117 seems to be saying "An Endorser provides claims about the root since there isn't a layer beneath the root layer that supplies Evidence. An Endorser also provides cryptographic material used by the root to authenticate the device and to sign evidence pertaining to upper / other layers."